### ISSUE TITLE
"normative" and "informative" terms not referenced

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag21#374]
Inherited from WCAG 2.0 - the glossary contains definitions of the terms "normative" and "informative", but those terms aren't referenced anywhere in the spec. It was probably meant to be referenced from the conformance section, but somehow that didn't happen. Do we want to add references to those terms, or just remove the terms? 

Note the Respec script uses the terms "normative" and "non-normative" rather than "normative" and "informative", to be aware of when we fix this.

**Comment by goodwitch** [comment]
Oh please do not remove these definitions from the glossary.  So many
people think it is all normative.  Those definitions are huge in helping
people understand what is truly required.

+1 to add references to those terms as appropriate.



glenda sims  |   team a11y lead   |    deque.com    |    512.963.3773
*web for everyone. web on everything.* -  w3 goals

[image: IAAP International Association of Accessibility Professionals:
Certified Professional in Accessibility Core Competencies (CPACC)]
<http://www.accessibilityassociation.org/certification>

**Comment by Ryladog** [comment]
+1 to Glenda

 

​​​​​* katie *

 

Katie Haritos-Shea 
Principal ICT Accessibility Architect, IAAP  <http://www.accessibilityassociation.org/recentlycertified> CPACC+WAS =  <http://www.accessibilityassociation.org/cpwacertificants> CPWA (WCAG/Section 508/ADA/AODA)

 

Cell: 703-371-5545 |  <mailto:ryladog@gmail.com> ryladog@gmail.com | Oakton, VA |  <http://www.linkedin.com/in/katieharitosshea/> LinkedIn Profile | Office: 703-371-5545 |  <https://twitter.com/Ryladog> @ryladog

NOTE: The content of this email should be construed to always be an expression of my own personal independent opinion, unless I identify that I am speaking on behalf of Knowbility, as their AC Rep at the W3C - and - that my personal email never expresses the opinion of my employer, Deque Systems.

**Comment by jake-abma** [comment]
+1 for Glenda

ps. maybe I missed the reason why, but I see the NOTE's are gone at the reference part https://w3c.github.io/wcag21/guidelines/ for normative and informative...

**Comment by joshueoconnor** [comment]
+1 to referencing them
